.golangci.yml

```diff
@@ -43,6 +43,12 @@ linters-settings:
   gocritic:
     disabled-checks:
       - ifElseChain
+  gofmt:
+    rewrite-rules:
+    - pattern:     'interface{}'
+      replacement: 'any'
```

Comment:
I wasn't convinced about this, but I wasn't up to date about the fact that Golang itself already made the transition internally, so 👍